modules/proc_configuring-notificationsconfiguration-crd-by-using-the-ocp-cli.adoc

```diff
@@ -0,0 +1,54 @@
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-notificationsconfiguration-crd-by-using-the-ocp-cli_{context}"]
+= Configuring NotificationsConfiguration (CR) by using the {OCP} CLI
+
+You can configure NotificationsConfiguration (CR) by using the {OCP} CLI.
+
+.Prerequisites
+
+* You have access to an {OCP} cluster with `cluster-admin` privileges.
+* You have installed the {gitops-title} Operator on your cluster.
+* You have enabled notifications for the Argo CD instance.
+
+.Procedure
+
+. Log in to the {OCP} cluster.
+
+. Run the following command to enable notifications:
++
+[source,yaml]
+----
+oc edit argocd openshift-gitops -n openshift-gitops
+----
+
+. Run the following command to edit the default `NotificationsConfiguration` (CR) in the cluster:
++
+[source,yaml]
+----
+oc edit notificationsconfiguration default-notifications-configuration -n openshift-gitops
+----
++
+where,
+
+- `default-notifications-configuration` - is the name of the default `NotificationsConfiguration` (CR)
+- `openshift-gitops` - is the name of the namespace
```

Comment:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks
```suggestion
where:

`default-notifications-configuration`:: The name of the default `NotificationsConfiguration` CR.
`openshift-gitops`:: The name of the namespace.
```

Reply:
Fixed.